### DISCUSSION TITLE
建議顯示毫秒

### DISCUSSION THREAD

**Discussion by Dong-Chen-1031** [SiongSng/cap-countdown#30] [提點子]
**### 建議顯示毫秒，這樣感覺更酷。**

**Comment by SiongSng** [comment]
@Dong-Chen-1031 關於您的建議我測試了一下，酷是酷，但是這種快速閃爍並不是非常舒服
我認為維持現有的每秒一次即可，毫秒太快了。

https://github.com/SiongSng/cap-countdown/assets/48402225/44dc57dd-47e3-4cf9-8618-a1c921ecff72

**Reply by Dong-Chen-1031** [reply]
好像也是，還是維持現狀好了，謝謝！😀
